### PR TITLE
Events Page - Get Involved Section

### PR DESCRIPTION
**Pull request by mirhamasala**
Style Events Page - Get Involved Section per Figma design [>>](https://www.figma.com/file/0pBfQs5tKI6bx5ypPIfVRA/FIL.org-V4-Prototypes?type=design&node-id=1231-33806&mode=design&t=Bzqh6wru2l2AsJtp-4)

Screenshot 2024-05-03 at 07 20 58

*Note:* Graphics to be added when design gets them.

### CODE REVIEW

File: src/app/events/page.tsx [hunk 1]
```diff
@@ -93,6 +96,22 @@ export default function Events() {
...
+        kicker="Get Involved"
+        title="Get in Touch With the Events Team"
+      >
+        <CardLayout>
+          {getInvolvedData.map(({ title, description, cta }) => (
+            <Card
+              key={title}
+              title={title}
+              description={description}
+              cta={{ ...cta }}
```
Comment (barbaraperic) [inline src/app/events/page.tsx:110]: why spread in an object? it's already an object?

File: src/app/events/data/getInvolvedData.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    cta: {
+      href: '#',
+      text: 'Submit Form',
+      icon: Clipboard,
+    },
+  },
+  {
+    title: 'Join Filecoin Orbit',
+    description:
+      'The Filecoin Orbit program promotes Filecoin technology through community events that raise network awareness. Apply to be an Orbit Ambassador.',
```
Comment (barbaraperic) [inline src/app/events/data/getInvolvedData.ts:28]: Orbit Ambassador should be a link?

Comment (mirhamasala) [reply src/app/events/data/getInvolvedData.ts:28]: The entire card is clickable - We shouldn't then also have links within the card. Something I already alerted design to. 🙏🏼

File: src/app/events/page.tsx (hunk 1 above)
Comment (mirhamasala) [reply src/app/events/page.tsx:110]: I guess some refactor gone wrong. 🙏🏼

**Pull request merged**